oak-run/src/main/java/org/apache/jackrabbit/oak/plugins/tika/TextExtractor.java

```diff
@@ -270,22 +272,24 @@ public InputStream get() {
             // not being present. This is equivalent to disabling
             // selected media types in configuration, so we can simply
             // ignore these errors.
-            log.debug("Failed to extract text from a binary property: {}."
+            String format = "Failed to extract text from a binary property: {}."
                             + " This often happens when some media types are disabled by configuration."
-                            + " The stack trace is included to flag some 'unintended' failures",
-                    path, e);
+                            + " The stack trace is included to flag some 'unintended' failures";
+            log.warn(format, linkageErrorFound ? path : new Object[]{path, e});
+            linkageErrorFound = true;
             parserErrorCount.incrementAndGet();
             return ERROR_TEXT;
         } catch (Throwable t) {
             // Capture and report any other full text extraction problems.
             // The special STOP exception is used for normal termination.
             if (!handler.isWriteLimitReached(t)) {
                 parserErrorCount.incrementAndGet();
-                parserError.debug("Failed to extract text from a binary property: "
-                        + path
+                String format = "Failed to extract text from a binary property: {}"
                         + " This is a fairly common case, and nothing to"
                         + " worry about. The stack trace is included to"
-                        + " help improve the text extraction feature.", t);
+                        + " help improve the text extraction feature.";
+                parserError.warn(format, throwableErrorFound ? path : new Object[]{path, t});
```

Comment:
Here I think we should use info, because "This is a fairly common case" in the message